apps/aehyperchains/src/aehc_connector.erl

```diff
@@ -1,36 +1,98 @@
 -module(aehc_connector).
 
--export([broadcast/4, verify/2, subscribe/2]).
+-export([send_tx/1, get_block/1]).
 
--type tx_hash() :: binary().
--type tx_payload() :: binary().
+-export([tx/3, header/2, block/2]).
 
--type account() :: <<_:256>>.
+-export([publish/2, subscribe/1]).
 
 -type connector() :: atom().
 
-%% TODO
--callback broadcast(From::account(), TxPayload::tx_payload(), To::account()) -> 
-    tx_hash().
+-callback send_tx(Tx::aetx:tx()) -> binary().
 
--callback verify(TxHash::tx_hash()) -> 
-    boolean().
+-callback get_block(Num::integer()) -> block().
```

Comment:
We also need to retrieve the blocks by hash. sed 's/Num/Height/g'

Reply:
+